winterfell/SETUP_WINTERFELL.md

```diff
@@ -1,58 +1,48 @@
 # Setting up Winterfell
 Winterfell is the server that handles all control logic in the application. This includes authentication, handling user information, it is basically the bulk of the main logistics.
 
-### Install Node
-This project is designed to work with node.js v5. 
+### Install Node Modules
+This project is designed to work with node.js v4.x.
 
 If you haven't already, we'll need to install node. Run the following at the same directory level as _package.json_.
+
 ```
+$ brew install node
 $ npm install
 ```
 
-### Install MongoDb No-SQL Database
-Refer to the [MongoDb installation page](https://docs.mongodb.com/master/installation/) to install.
-We used Homebrew to install Mongo.
-```
-$ brew install mongodb
-```
+### Startup Vagrant
```

Comment:
Please include a short blurb with how to Download Vagrant and virtualbox + any other things to set it up correctly before moving on to "vagrant up"